recipes/epa-ng/conda_build_config.yaml

```diff
@@ -0,0 +1,6 @@
+c_compiler:
+  - gcc      # [linux]
+  - clang    # [osx]
+cxx_compiler:
+  - gxx      # [linux]
+  - clangxx  # [osx]
```

Comment:
I presume it doesn't work without this file?

Reply:
Yes this resulted from extensive troubleshooting, which you can read about here: https://forum.qiime2.org/t/issue-with-cmake-when-making-epa-ng-conda-recipe/6430/9